types/lib-hubs.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module "lib-hubs/packages/three-particle-emitter/lib/esm/index" {
```

Comment:
Not worth spending very much time on. But lib-hubs is actually written in typescript and has types https://github.com/MozillaReality/lib-hubs/blob/master/packages/three-particle-emitter/lib/types/index.d.ts I wonder if we can get that to work somehow.

Reply:
I'd have used those types as is but we are not exposing the material property there so I just preferred to add the d.ts file that updating that in the external module (considering also that we will eventually move away from this implementation)